index.src.html

```diff
@@ -974,21 +973,21 @@ <h3 id="strip-url">
       return <code>no referrer</code>.
     </li>
     <li>
-      Set <var>url</var>'s <a>username</a> to the empty string.
+      Set <var>url</var>'s <a for=url>username</a> to the empty string.
     </li>
     <li>
-      Set <var>url</var>'s <a>password</a> to <code>null</code>.
+      Set <var>url</var>'s <a for=url>password</a> to the empty string.
     </li>
     <li>
-      Set <var>url</var>'s <a>fragment</a> to <code>null</code>.
+      Set <var>url</var>'s <a for=url>fragment</a> to <code>null</code>.
     </li>
     <li>
       If the <code><a>origin-only flag</a></code> is <code>true</code>,
       then:
 
       <ol>
         <li>
-          Set <var>url</var>'s <a for=url>path</a> to <code>null</code>.
+          Set <var>url</var>'s <a for=url>path</a> to « the empty string ».
```

Comment:
Should this be the empty list?

Reply:
Hmm, it looks like it. I was on autopilot looking at https://github.com/w3c/webappsec-referrer-policy/issues/153#issuecomment-864359547.

Reply:
Done. I've made this `« »`, but let me know if you think it should be "an empty list".